distributed/diagnostics/tests/test_scheduler_plugin.py

```diff
@@ -138,8 +138,9 @@ async def close(self):
             self.history.append("closed")
 
     plugin = LifeCycle()
-    async with Scheduler(plugins=[plugin], dashboard_address=":0") as s:
-        pass
+    async with Scheduler(dashboard_address=":0") as s:
+        s.add_plugin(plugin)
+        await plugin.start(s)
```

Comment:
#5348 removes the plugins= parameter from Scheduler and Worker. It was exclusively used here.